feg/gateway/services/aaa/base_acct/heartbeat_reset_go1_15.go

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//go:build go1.15
+// +build go1.15
```

Comment:
does it mean that package `base_acct` is going to be built with go.1.15?

Reply:
It just ticker reset capability, it didn't exist prior to Go 1.15, so the reset call for older compilers will be noop